Feature/time sheets

mani9694 opened the pull request.

gopibabus commented on app/Http/Controllers/TimesheetsController.php:
```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Controllers;
+
+class TimesheetsController extends Controller
+{
+    /**
+     * Create a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+
+    /**
+     * Show the application dashboard.
+     *
```
Please remove this file. Requested changes should be part of existing page.

gopibabus commented on resources/views/timesheets.blade.php:
```diff
@@ -0,0 +1,6 @@
+@extends('layouts.app')
+
+@section('content')
+    <div class="container jumbotron">
+        <h1 class="display-4">Welcome to Timesheets</h1>
+@endsection
```
Please remove this file. View is not required at this point.

gopibabus commented on routes/web.php:
```diff
@@ -20,3 +20,4 @@
 Auth::routes();
 
 Route::get('/home', [App\Http\Controllers\HomeController::class, 'index'])->name('home');
+Route::get('/time-sheets', [App\Http\Controllers\TimesheetsController::class, 'index']);
```
Remove this route. Route is not required at this point.

mani9694 commented:
Deleted not need files and updated the PR.
This is how it looks in my local.
<img width="1679" alt="Screen Shot 2020-12-06 at 6 59 39 PM" src="https://user-images.githubusercontent.com/33164087/101298845-7850ab80-37f5-11eb-9d66-7313ea889af0.png">

gopibabus commented on resources/views/layouts/app.blade.php:
```diff
@@ -27,6 +27,9 @@
                 </a>
+                <a class="navbar-brand" href="{{ url('/') }}">
+                    {{ 'Time sheets' }}
+                </a>
```
@mani9694 we should restrict this menu item for only logged in users. I am applying that fiix.
